js/apps/ProjectPanel/views/PanelRenderer.js

```diff
@@ -40,6 +40,13 @@ SPM.PanelRenderer = {
             this.addTitle(SPM.Utils.getDueDate(this.project.due), this.project.name);
         }
 
+        if (this.project.errors && this.project.errors.moreThanOneTrelloCard) {
+            var projects = _.reduce(this.project.errors.moreThanOneTrelloCard, function(memo, project) {
+                return memo + ' - '+ project.name;
+            }, "");
+            this.addError('Plusieurs projets pointent vers cette discussion Slack: ' + projects);
```

Comment:
C'est problématique de forcer ça comme erreur. Dans le cas ou un projet a plusieurs itération, donc plusieurs carte trello, il garde cependant le même chan. Je prefererai le garder comme alerte très soft (voir pas du tout), et prendre à chaque fois le plus récent. Et quand on y réfléchit, c'est assez logique.